.github/workflows/ci.yml

```diff
@@ -1,6 +1,7 @@
+name: CI
+
 on:
   push:
-  pull_request:
```

Comment:
Removed the `pull_request` one as it would trigger our CI jobs twice which isn't needed.